py/desiutil/test/test_io.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""Test desiutil.io.
+"""
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+# The line above will help with 2to3 support.
+import unittest
+import sys
+import numpy as np
+from warnings import catch_warnings, simplefilter
```

Comment:
You're importing from warnings but not actually using these as far as I can tell.